lms/djangoapps/ccx/views.py

```diff
@@ -410,7 +410,7 @@ def ccx_invite(request, course, ccx=None):
     action = request.POST.get('enrollment-button')
     identifiers_raw = request.POST.get('student-ids')
     identifiers = _split_input_list(identifiers_raw)
-    auto_enroll = True if 'auto-enroll' in request.POST else False
+    auto_enroll = True
```

Comment:
Do you actually need to make this change? I know the control is disabled, but isn't the value still sent in the POST?